src/test/suite/fixtures/recorded/languages/rust/changeNameTrap.yml

```diff
@@ -0,0 +1,38 @@
+languageId: rust
+command:
+  spokenForm: change name trap
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: t}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: name}
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: |
+    enum Maybe<T> {
+        Just(T),
```

Comment:
Likewise, not sure why this one is name?  If anything maybe "callee"?

Reply:
Enum variants look enough like structs themselves that I think name makes the most sense for consistency.

Reply:
Just read up on Rust enums; yep makes sense 👍